aws/resource_aws_ssm_parameter.go

```diff
@@ -67,6 +67,10 @@ func resourceAwsSsmParameter() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"data_type": {
```

Comment:
We can provide plan-time validation for this new attribute by including the following in the schema definition for it. 😄 

```go
ValidateFunc: validation.StringInSlice([]string{
	"aws:ec2:image",
	"text",
}, false),
```